scripts/create_testenv.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -ex # fail on first error, print commands
+
+command -v conda >/dev/null 2>&1 || {
+  echo "Requires conda but it is not installed.  Run install_miniconda.sh." >&2;
+  exit 1;
+}
+
+ENVNAME="testenv"
+PYTHON_VERSION=${PYTHON_VERSION:-3.7} # if no python specified, use 3.7
```

Comment:
If we're going to use TensorFlow then Python 3.7 could be an issue

Reply:
5:4 in favor of pytorch actually (my vote was late)